### PR TITLE
Pgadmin northwind psql

### PR DESCRIPTION
**Pull request by Thedand**

### CODE REVIEW

**Comment by pthom** [comment]
Hello, 

Thanks for this PR. Could you give me some more details via a short description in this PR?

Questions:
- I tested pgAdmin. It brings value and seems OK (except for a point about the dbdata volume location: see my code review)
- The modifications due to Pycharm seem to relate to something else; and only work with Pycharm professional (which is not free, and which I could not test). I'd rather only add what is related to pdAdmin.

Thanks!

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Northwind database for Postgres
 
 A simple sql script that will populate a database with the famous northwind example, adapted for postgres.
+Two connections PgAdmin and PyCharm/
```
Comment (pthom) [inline README.md:4]: Please remove what is PyCharm related

File: README.md [hunk 2]
```diff
@@ -19,41 +20,70 @@ Use the provided sql file `nortwhind.sql` in order to populate your database.
...
+ - Name: db
+ - Host name/address: db
+ - Username: northwind_user
+ - Password: thewindisblowing
+
+- Select DB - northwind
+
+
+#### 4. Connect PyCharm Professional Edition
+
```
Comment (pthom) [inline README.md:75]: this works only with Pycharm Professional (I guess), and should be part of a separate PR.

File: docker-compose.yml [hunk 3]
```diff
@@ -2,11 +2,39 @@ version: '3'
...
-    image: postgres:12
+    container_name: db
+    image: postgres:13
     environment:
       POSTGRES_DB: northwind
       POSTGRES_USER: northwind_user
       POSTGRES_PASSWORD: thewindisblowing
     volumes:
-      - ./dbdata:/var/lib/postgresql/data
+      - /var/lib/postgresql/data:/var/lib/postgresql/data
```
Comment (pthom) [inline docker-compose.yml:12]: It will be better to leave the volume location as local subfolder as before:

````
volumes:
      - ./dbdata:/var/lib/postgresql/data
````

i.e, the folder is kept locally. Otherwise, bad things may happen:
* the docker container my overwrite your own main postgres database folder
* if the `/var/lib/postgresql/data` folder does not exist (for example, on a Mac; such a my computer), it will also fail

File: docker-compose.yml [hunk 4]
```diff
@@ -2,11 +2,39 @@ version: '3'
...
+      PGADMIN_DEFAULT_PASSWORD: admin
+      PGADMIN_LISTEN_PORT: 5050
+      PGADMIN_CONFIG_SERVER_MODE: 'False'
+    volumes:
+      - pgadmin:/root/.pgadmin
+    ports:
+      - "5050:5050"
+    networks:
+      - db
+
```
Comment (pthom) [inline docker-compose.yml:33]: This works nicely

**Review by pthom — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -19,47 +20,61 @@ Use the provided sql file `nortwhind.sql` in order to populate your database.
...
+- Create - Server
+ - Name: db
+ - Host name/address: db
+ - Username: northwind_user
+ - Password: thewindisblowing
+
+- Select DB - northwind
+
+#### 5. Stop docker-compose
```
Comment (pthom) [inline README.md:73]: This should be  number 4.

File: docker-compose.yml (hunk 3 above)
Comment (pthom) [reply docker-compose.yml:12]: Hum, you marked this as solved, but did  not modify anything. Is this a mistake, or do you have an explanation for this?

**Review by pthom — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 3 above)
Comment (Thedand) [reply docker-compose.yml]: I'm sorry, made changes.

**Review by pthom — APPROVED** [review]
